portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFileEntryLocalServiceImpl.java

```diff
@@ -361,6 +361,10 @@ public DLFileEntry addFileEntry(
 			DLStoreUtil.addFile(dlStoreRequest, inputStream);
 		}
 
+		if (_log.isDebugEnabled()) {
+			_log.debug("Adding empty file " + fileName, new Exception());
```

Comment:
Why is the exception needed? 

You could use com.liferay.portal.kernel.log.Log#debug(java.lang.Object)

Reply:
@robertoDiaz  To show the stacktrace. DXP cloud needs to know who (or how) is originally generating this kind of empty files. If I don't add the stacktrace they will just know that they have this problem (that is more than now, of course), but I find it useful to print the stacktrace.

What do you think?